sponsorblock.lua

```diff
@@ -82,11 +82,14 @@ if legacy then
 end
 
 local utils = require "mp.utils"
-scripts_dir = mp.find_config_file("scripts")
+local scripts_dir = mp.find_config_file("scripts")
+local home_dir = os.getenv("HOME")
+local data_dir = os.getenv("XDG_DATA_HOME") or home_dir and utils.join_path(home_dir, ".local/share") or scripts_dir
```

Comment:
please don't make it fallback to home, maybe we can do something like this instead https://github.com/hishamhm/datafile/blob/master/datafile/openers/unix.lua